objectbox/lib/src/native/query/query.dart

```diff
@@ -725,6 +729,105 @@ class Query<T> {
     return result;
   }
 
+  /// Finds Objects matching the query.
+  /// TODO rename to just `stream`
```

Comment:
TODO is visible in public docs, intentional?

Reply:
won't be when it's resolved before merging this PR